rust/Makefile.am

```diff
@@ -84,7 +87,8 @@ vendor:
 
 if HAVE_CBINDGEN
 gen/rust-bindings.h: $(RUST_SURICATA_LIB)
-	cbindgen --config $(abs_top_srcdir)/rust/cbindgen.toml \
+	cd $(abs_top_srcdir)/rust && \
+		cbindgen --config $(abs_top_srcdir)/rust/cbindgen.toml \
 		--quiet --verify --output $(abs_top_builddir)/rust/gen/rust-bindings.h || true
```

Comment:
This is actually not related, but needs to be done in master as part of the other PR that change the dist pattern for Rust.